gridt/src/app/home/home.page.html

```diff
@@ -59,9 +59,24 @@ <h3>
                         </h3>
                         <p>{{ isLeaderDone(leader, movement) ? 'Done!' : 'Pending...' }}</p>
                       </div>
-                      <small *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message">
-                        {{ leader.last_signal.message }}
-                      </small>
+                      <div *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message" id="expandable">
+                        <app-expandable signalMessage={{leader.last_signal.message}}></app-expandable>
+                        <!-- <ion-button fill="clear" (click)="showFullMessage(leader.last_signal.message)">
+                          <div class="leader-message">
+                            <small>
+                              {{ leader.last_signal.message }}
+                            </small>
+                          </div>
+                        </ion-button> -->
+                        <!-- <ion-text *ngIf="!itemExpanded">
+                          <p class="overflow-elipsis">{{ leader.last_signal.message }}</p>
+                        </ion-text>
+                          <app-expandable [signalMessage]="leader.last_signal.message">
+                            <ion-text *ngIf="itemExpanded">
+                              <p class="overflow-visible">{{ leader.last_signal.message }}</p>
+                            </ion-text>
+                          </app-expandable> -->
```

Comment:
General rule of thumb: never commit commented code. It serves no purpose as your history is already captured by git.